### PR TITLE
SOMMELIER-13

### PR DESCRIPTION
**Pull request by Mixxx-a**
Added SearchInput and Image tests, need to solve problems with Image before merging

### CODE REVIEW

File: App/components/Image/Image.story.tsx [hunk 1]
```diff
@@ -16,7 +14,33 @@ const defaultImage = () => (
...
+    //       light: 'light',
+    //       lightGray: 'lightGray',
+    //       primary: 'primary',
+    //       lightGrayBorder: 'lightGrayBorder',
+    //       grayBorder: 'grayBorder',
+    //     },
+    //     'grayBorder'
+    //   ),
+    //   borderWidth: number('borderWidth', 1),
+    // }}
```
Comment (constantkek) [inline App/components/Image/Image.story.tsx]: Commented code from sommelier-12 (actually my branch)))). Now 12th fixed, merge with development will fix that (or just erase here)

File: App/components/SearchInput/input.story.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+const StyledPadding = styled.View`
+  margin-top: 5px;
+  margin-bottom: 5px;
+`
+
+export const list = () => (
+  <StyledWrapper>
+    <SearchInput
+      onChange={() => 's'}
```
Comment (constantkek) [inline App/components/SearchInput/input.story.tsx:23]: here and after that, dont catch what does that means, but probably like a 'noop' function. Maybe onChange is optional?

Comment (Mixxx-a) [reply App/components/SearchInput/input.story.tsx:23]: All questions to @Maximus135

**Comment by Mixxx-a** [comment]
image
image

Need to sort out the fireEvent function to test the onChange() in SearchInput

**Pull request merged**